### PR TITLE
Cloud release notes February

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-2239

### CODE REVIEW

**Comment by jramos-crl** [comment]
@gemma-shay this is set to go live on Feb 7, it looks like? If so, we should have the decoupled UI released by then if you want to include a line for it.

**Review by thtruo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12874)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @gemma-shay, @jramos-crl, and @kevin-v-ngo)

___
*[releases/cloud.md, line 20 at r1](https://reviewable.io/reviews/cockroachdb/docs/12874#-MuWZ8E7ENkfV80mVxJm:-MuWZ8E7ENkfV80mVxJn:blx5kqj) ([raw file](https://github.com/cockroachdb/docs/blob/559918e2c7005d41cdce9e6ceef828377f337091/releases/cloud.md#L20)):*
> ```Markdown
> ## February 7, 2022
> 
> <h3>Console changes</h3>
> ```

@gemma-shay Do we have a policy around whether or not to announce beta functionality? FWIW we just made the Datadog metrics integration available for select customers. It's behind a feature flag. Interested customers can request to be put on the beta.



_Code quote:_
```Markdown
Console changes
```

**Comment by gemma-shay** [comment]
@thtruo Up to you! I usually wouldn't document it until it's fully available, but if you're looking for more beta users I think it could be helpful to make a note.

**Comment by gemma-shay** [comment]
@jramos-crl What's our best guess eta? If it's early this coming week, I was thinking of giving decoupled hardware its own release note. Working on changing the protocol here so major feature flags don't have to wait for monthly notes.

**Comment by jramos-crl** [comment]
@gemma-shay it will probably be mid-next week or so at this point. That's a good call regarding having its own release notes. Let's do that then 👍

**Comment by thtruo** [comment]
Ack that makes sense @gemma-shay in this case, no need to mention the beta version of this for the Feb release.

**Review by kevin-v-ngo — APPROVED** [review]

**Comment by gemma-shay** [comment]
Hi @emily-horing, adding you because I just saw that new regions were added for Serverless. @ianjevans do you mind doing a quick review of the regions addition as well?

File: releases/cloud.md [hunk 1]
```diff
@@ -15,6 +15,29 @@ Get future release notes emailed to you:
...
+    
+    GCP                              | AWS
+    ---------------------------------|------------
+    Oregon (`us-west2`)              | Mumbai (`ap-south-1`)
+    Sao Paulo (`southamerica-east1`) | Frankfurt (`eu-central-1`)
+    South Carolina (`us-east1`)      | N. Virginia (`us-east-1`)
+
+<h3>Console changes</h3>
+
+- The [**Terminate session** and **Terminate Statement**](../cockroachcloud/sessions-page.html#sessions-list) options are now enabled for {{ site.data.products.db }} clusters running CockroachDB [v21.2.2](v21.2.2.html) or later.
```
Comment (ianjevans) [inline releases/cloud.md:32]:
```suggestion
- The [**Terminate Session** and **Terminate Statement**](../cockroachcloud/sessions-page.html#sessions-list) options are now enabled for {{ site.data.products.db }} clusters running CockroachDB [v21.2.2](v21.2.2.html) or later.
```

File: releases/cloud.md [hunk 2]
```diff
@@ -15,6 +15,29 @@ Get future release notes emailed to you:
...
+- Selecting a transaction from the [**Transactions** page](../cockroachcloud/transactions-page.html) now opens a new [**Transaction details**](../cockroachcloud/transactions-page.html#transaction-details-page) page with an improved design.
```
Comment (ianjevans) [inline releases/cloud.md:33]:
```suggestion
- Selecting a transaction from the [**Transactions** page](../cockroachcloud/transactions-page.html) now opens a new [**Transaction Details**](../cockroachcloud/transactions-page.html#transaction-details-page) page with an improved design.
```

**Review by ianjevans — CHANGES_REQUESTED** [review]

**Review by emily-horing — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12874)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @gemma-shay and @jramos-crl)

___
*[releases/cloud.md, line 20 at r1](https://reviewable.io/reviews/cockroachdb/docs/12874#-MuWZ8E7ENkfV80mVxJm:-MvKDEQN2TO1_fHXf8bG:b87t5z5) ([raw file](https://github.com/cockroachdb/docs/blob/559918e2c7005d41cdce9e6ceef828377f337091/releases/cloud.md#L20)):*
<details><summary><i>Previously, thtruo (Tommy) wrote…</i></summary><blockquote>

@gemma-shay Do we have a policy around whether or not to announce beta functionality? FWIW we just made the Datadog metrics integration available for select customers. It's behind a feature flag. Interested customers can request to be put on the beta.
</blockquote></details>

cc @thtruo for datadog specifically. In general I think it's a nice way to preview functionality and get more possible beta testers to announce closed betas in notes. just my 2 cents.

**Review by emily-horing — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12874#-:-MvKDbVy7Z9BcCYonQm9:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @gemma-shay and @jramos-crl)

**Comment by gemma-shay** [comment]
@emily-horing @thtruo Ack, I'm going to merge this and follow up with Tommy about adding a Datadog note.

**Comment by thtruo** [comment]
Had a followup convo with @gemma-shay - I'm cool if we exclude Datadog in the release note and instead make a mention of it in the marketing emails that go out calling for beta testers

**Comment by emily-horing** [comment]
I'm just realizing (too late) that we seem to have excluded a note about faster perf for autogenerated passwords and the option to autogenerate passwords from the SQL user creation form. Unless that was covered in the prior notes? I thought it was a Jan change though. cc @JeffSwenson

**Pull request merged**